### PR TITLE
fix url.src for genet

### PR DESCRIPTION
**Pull request by zapashcanon**
@zoggy fyi

### CODE REVIEW

**Comment by zoggy** [comment]
I think the urls should be using github.io instead of github.com .

File: packages/genet/genet.0.1/opam [hunk 1]
```diff
@@ -26,6 +26,6 @@ depends: [
 install: [make "install"]
 synopsis: "Genet is tool to build a continuous integration platform."
 url {
-  src: "http://zoggy.github.com/genet/genet-0.1.tar.gz"
+  src: "https://opam.ocaml.org/cache/md5/80/807a4a297ba32a3a177b6ddf04d1a141"
```
Comment (dra27) [inline packages/genet/genet.0.1/opam:29]:
```suggestion
  src: "https://zoggy.github.io/genet/genet-0.1.tar.gz"
```

File: packages/genet/genet.0.2/opam [hunk 2]
```diff
@@ -26,6 +26,6 @@ depends: [
 install: [make "install"]
 synopsis: "Genet is tool to build a continuous integration platform."
 url {
-  src: "http://zoggy.github.com/genet/genet-0.2.tar.gz"
+  src: "https://opam.ocaml.org/cache/md5/8a/8a839c8fe8e0ab99abfaea1d45d3fea0"
```
Comment (dra27) [inline packages/genet/genet.0.2/opam:29]:
```suggestion
  src: "https://zoggy.github.io/genet/genet-0.2.tar.gz"
```

File: packages/genet/genet.0.1/opam [hunk 3]
```diff
@@ -26,6 +26,6 @@ depends: [
 install: [make "install"]
```
Comment (mseri) [inline packages/genet/genet.0.1/opam:26]:
```suggestion
install: [make "install"]
license: "LGPL-3.0-only"
```

File: packages/genet/genet.0.2/opam [hunk 4]
```diff
@@ -26,6 +26,6 @@ depends: [
 install: [make "install"]
```
Comment (mseri) [inline packages/genet/genet.0.2/opam:26]:
```suggestion
install: [make "install"]
license: "LGPL-3.0-only"
```

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**